osu.Game.Rulesets.Osu.Tests/TestSceneSpinner.cs

```diff
@@ -17,32 +17,54 @@ public class TestSceneSpinner : OsuSkinnableTestScene
     {
         private int depthIndex;
 
-        public TestSceneSpinner()
+        private TestDrawableSpinner drawableSpinner;
+
+        [TestCase(false)]
+        [TestCase(true)]
+        public void TestVariousSpinners(bool autoplay)
         {
             AddStep("Miss Big", () => SetContents(() => testSingle(2)));
             AddStep("Miss Medium", () => SetContents(() => testSingle(5)));
             AddStep("Miss Small", () => SetContents(() => testSingle(7)));
```

Comment:
Seeing the `[TestCase]` attributes added on this method, did you mean to remove these and replace them with the `false` test case? As it stands these are covered doubly.